src/ast2ram/AstToRamTranslator.cpp

```diff
@@ -439,7 +431,7 @@ Own<ram::Statement> AstToRamTranslator::translateNonRecursiveRelation(
     VecOwn<ram::Statement> res;
 
     // the ram table reference
-    Own<ram::RelationReference> rrel = translateRelation(&rel);
+    std::string rrel = translateRelation(&rel);
```

Comment:
This variable name was a little unhelpful when it was a relation, and even less so now. Rename it to something like `relationName`.

The comment on the line above seems outdated, too.

Reply:
Indeed, we should rename this variable to relName.